qiskit/qpy/formats.py

```diff
@@ -360,6 +360,10 @@
 
 # EXPR_TYPE
 
+EXPR_TYPE_FLOAT = namedtuple("EXPR_TYPE_FLOAT", [])
+EXPR_TYPE_FLOAT_PACK = "!"
+EXPR_TYPE_FLOAT_SIZE = struct.calcsize(EXPR_TYPE_FLOAT_PACK)
+
 EXPR_TYPE_DISCRIMINATOR_SIZE = 1
```

Comment:
super super minor: can you put the discriminator first, since that's common to all things of `EXPR_TYPE`?